templates/cinder/bin/init.sh

```diff
@@ -26,51 +26,54 @@ export DBPASSWORD=${DatabasePassword:?"Please specify a DatabasePassword variabl
 export PASSWORD=${CinderPassword:?"Please specify a CinderPassword variable."}
 export TRANSPORTURL=${TransportURL:-""}
 
-export CUSTOMCONF=${CustomConf:-""}
-
+DEFAULT_DIR=/var/lib/config-data/default
+CUSTOM_DIR=/var/lib/config-data/custom
+MERGED_DIR=/var/lib/config-data/merged
 SVC_CFG=/etc/cinder/cinder.conf
```

Comment:
Not if we retain L41. I suppose it could be eliminated and just use the "/etc/cinder/cinder.conf" string in L41. The shell variable was referenced a lot more before this patch, so having a variable made sense. Given my vote to retain L41, what are your thoughts on retaining the variable?

Reply:
I'm ok if we change it to:
```
 SVC_SAMPLE_CFG=/etc/cinder/cinder.conf.sample
```

Reply:
Sure, that sounds good. In addition, we could create a 1-line cinder.conf file that's just a comment directing people to the cinder.conf.d directory. That might be useful because RHOSP hasn't traditionally used that directory, and people may wonder what happened to the cinder.conf file they're used to seeing.

Thoughts?

Reply:
I think it's a great idea.